tests/unit_tests/test_wave_06.py

```diff
@@ -194,16 +207,22 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is falsy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories
+    assert result == False
+    assert len(tai.inventory)  == 3
+    assert len(jesse.inventory) == 3
+    assert item_d and item_e and item_f in jesse.inventory
+    assert item_a and item_b and item_c in tai.inventory
```

Comment:
You can combine lines 219 and 220 if you want:
```python
assert len(tai.inventory) == 3 and len(jesse.inventory) == 3
```

You can use list destructing and write lines 221-222 lines like this too - a pythonic way of doing it:
```python
assert [item_d, item_e, item_f] == jesse.inventory
assert [item_a, item_b, item_c] == tai.inventory
```

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538